src/jsTest/kotlin/com/autodesk/coroutineworker/Platform.kt

```diff
@@ -0,0 +1,10 @@
+package com.autodesk.coroutineworker
+
+import kotlin.coroutines.CoroutineContext
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.promise
+
+actual fun <T> runBlocking(context: CoroutineContext, block: suspend CoroutineScope.() -> T): dynamic = GlobalScope.promise { block() }
+
+actual fun Any.ensureNeverFrozen() = asDynamic()
```

Comment:
Why `asDynamic()`. will the following not work?
```suggestion
actual fun Any.ensureNeverFrozen() = this
```

Reply:
this function return Unit so `this` won't work

Reply:
ah! right. why not just ` = Unit` then?

Reply:
it can be Unit indeed, but actually, when i'm implementing the library as this into another project, the other projects builds and everything is working fine in execution.
but the jsTest always fails as i mentioned the tests should be rewritten to support the javascript coroutines nature, i also put an example link.
so i used asDynamic() for for javascript convenience

Reply:
Right for js tests, I understand for runBlocking why we need to use dynamic, but for this function that returns nothing, i think it's okay to just return `Unit`, no? Sorry. I'm not sure I understand the issue

Reply:
yes it's ok to return Unit, i will that commit now

Reply:
Updated the suggestion to be `= Unit`